src/sinks/vector/v2/config.rs

```diff
@@ -76,9 +79,9 @@ impl VectorConfig {
             .clone()
             .map(|uri| uri.uri)
             .unwrap_or_else(|| uri.clone());
-        let healthcheck_client = VectorService::new(client.clone(), healthcheck_uri);
+        let healthcheck_client = VectorService::new(client.clone(), healthcheck_uri, false);
```

Comment:
Should we use the compression setting here too?

Reply:
Being a small message I can't see any gain. Tonic should send plaintext any message that compressed result is larger than the plaintext (I havent checked) with only a waste in CPU. I think disabled is wiser.

Reply:
Ah, yeah, that's a good point. 👍